lib/guardian_db/token.ex

```diff
@@ -5,19 +5,15 @@ defmodule GuardianDb.Token do
 
   use Ecto.Schema
   import Ecto.Changeset
-  import Ecto.Query, only: [from: 2]
+  import Ecto.Query, only: [where: 2]
 
+  alias Ecto.Query
```

Comment:
Is this alias necessary given the import above?

Reply:
@doomspork yeah `credo` bitching about it